Content.Shared/Sound/SoundSpecifier.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using Content.Shared.Audio;
+using Robust.Shared;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Sound
+{
+    [DataDefinition]
+    public abstract class SoundSpecifier
+    {
+        public abstract string GetSound();
+    }
+
+    [DataDefinition]
+    public class SoundEmptySpecifier : SoundSpecifier
+    {
+        public override string GetSound()
+        {
+            return string.Empty;
+        }
+    }
```

Comment:
You might as well merge the behavior of this and SoundPathSpecifier by making it properly nullable while keeping required: true.

Reply:
done